Tools/cmake/swigconfig.h.in

```diff
@@ -80,13 +82,11 @@
 #define SWIG_LIB_WIN_UNIX "C:/cygwin/usr/local/@SWIG_LIB@"
```

Comment:
SWIG_LIB_WIN_UNIX is hard coded here and needs to replicate the logic in https://github.com/swig/swig/blob/ac8de714af1d7fbd755597b0f51792212f552ef7/configure.ac#L2745-L2751 so that the logic in https://github.com/swig/swig/blob/ac8de714af1d7fbd755597b0f51792212f552ef7/Source/Modules/main.cxx#L946-L947 works.